src/Umbraco.Examine.Lucene/BackOfficeExamineSearcher.cs

```diff
@@ -146,6 +148,12 @@ public IEnumerable<ISearchResult> Search(
                     ? currentUser.CalculateContentStartNodeIds(_entityService, _appCaches)
                     : Array.Empty<int>();
                 AppendPath(sb, UmbracoObjectTypes.Document, allContentStartNodes, searchFrom, ignoreUserStartNodes, _entityService);
+
+                if (trashed.HasValue)
+                {
+                    AppendRequiredTrashPath(trashed.Value, sb);
+                }
+
```

Comment:
Should this apply to media as well? Seems reasonable that one would expect it to work for all entity types that support trashing?

Reply:
Good point, implemented it for media too :)